labelbox/schema/export_params.py

```diff
@@ -34,3 +36,18 @@ class CatalogExportParams(DataRowParams):
 class ModelRunExportParams(DataRowParams):
     # TODO: Add model run fields
     pass
+
+
+def validate_array_length(array, max_length, array_name):
+    if len(array) > max_length:
+        raise ValueError(f"{array_name} cannot exceed {max_length} items")
+
+
+def validate_catalog_export_params(params: CatalogExportParams):
+    if "model_run_ids" in params and params["model_run_ids"] is not None:
```

Comment:
`params["model_run_ids"] is not None` can just be `params["model_run_ids"]`, same with the line below

Reply:
🤔 that's how I started and it was causing issues, I'll revisit to see if that's expected